console/src/components/Services/Data/TableModify/ColumnEditor.js

```diff
@@ -71,6 +75,25 @@ const ColumnEditor = ({
     dispatch(editColumn(colName, 'isUnique', e.target.value === 'true'));
   };
 
+  const getColumnAliasInput = () => {
+    if (!supportAliasing) return;
+    return (
+      <div className={`${styles.display_flex} form-group`}>
+        <label className="col-xs-2">Custom GraphQL field</label>
```

Comment:
This seems little weird as other columns are one liner and this extends to 3 lines

Reply:
Fixed.